index.html

```diff
@@ -290,7 +291,42 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
-
+    <footer>
+      <form
+        action="https://formspree.io/f/xdoyydwq"
+        method="POST"
+        class="form-container"
+      >
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <div class="inputs">
+          <div class="group-input">
+            <div class="fname">
+   
+              <input type="text" placeholder="First Name" />
+            </div>
+            <div class="lname">
+          
+              <input type="text" placeholder="Last Name"/>
```

Comment:
- [ ] The inputs should have `required` attribute to activate he validation on submit. I can still submit the form even when some of the fields are empty, let's fix this by adding the `required` attribute to validate our form such that users can't submit empty forms.